controllers/dspipeline_controller.go

```diff
@@ -501,6 +507,46 @@ func (r *DSPAReconciler) PublishMetrics(dspa *dspav1alpha1.DataSciencePipelinesA
 	}
 }
 
+func (r *DSPAReconciler) GetComponents(ctx context.Context, dspa *dspav1alpha1.DataSciencePipelinesApplication) dspav1alpha1.ComponentStatus {
+	log := r.Log.WithValues("namespace", dspa.Namespace).WithValues("dspa_name", dspa.Name)
+	log.Info("Updating components endpoints")
+
+	envoyUrl, err := r.GetEnvoyServiceHostname(ctx, dspa)
+	if err != nil {
+		log.Info(err.Error())
```

Comment:
these are errors, so use log.Error

```suggestion
		log.Error(err, "unable to get mlmd envoy proxy's service hostname")
```